StringCalculator.Tests/StringCalculatorTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿using NUnit.Framework;
```

Comment:
General comment on your commits - they need to be atomic and more frequent. Atomic commits are smaller commits that have a specific feature/piece of work. Because you'll be breaking down your commits, they will need to be more frequent (i.e. there will be more of them, and that's okay)